homeassistant/components/neato/sensor.py

```diff
@@ -81,6 +81,9 @@ def available(self):
     @property
     def state(self):
         """Return the state."""
+        if not self._state:
+            # Not yet updated
+            return
```

Comment:
Explicitly return `None`

```suggestion
            return None
```

Reply:
Done, thanks. I've thought about this a second time and changed `self._available` in order to be false until the first update. What do you think? Seems much cleaner to me. Should we still return `None` if `self._state` is not there?